src/Application/FileSystem/State.elm

```diff
@@ -99,6 +107,35 @@ gotDirectoryList json model =
             )
 
 
+gotItemUtf8 : { pathSegments : List String, text : String } -> Manager
+gotItemUtf8 { pathSegments, text } model =
+    (case model.sidebar of
+        Just sidebar ->
+            case sidebar.mode of
+                Sidebar.EditPlaintext _ ->
+                    { model
+                        | sidebar =
+                            Just
+                                { sidebar
+                                    | mode =
+                                        Sidebar.EditPlaintext
+                                            (Just
+                                                { text = text
+                                                , originalText = text
+                                                }
+                                            )
+                                }
+                    }
```

Comment:
Can we use some pipes here to avoid the indentation?
```elm
{ text
, originalText
}
    |> Just
    |> Sidebar.EditPlaintext
    |> (\mode -> { sidebar | mode = mode })
    |> Just
    |> (\sidebar -> ...)
```

Reply:
Same goes for the `src/Application/Drive/State/Sidebar.elm` module.

Reply:
Ah! I didn't think of this! It was bothering me, too.